config/manager/webhook.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: webhook
+  namespace: system
+  labels:
+    control-plane: webhook
+spec:
+  selector:
+    matchLabels:
+      control-plane: webhook
+  template:
+    metadata:
+      labels:
+        control-plane: webhook
+    spec:
+      nodeSelector:
+        cray.nnf.manager: "true"
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - command:
+        - /manager
+        args:
+        - --mode=webhook
+        image: controller:latest
+        name: webhook
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 30
+          periodSeconds: 25
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 8081
+          initialDelaySeconds: 15
+          periodSeconds: 10
+        resources:
+          requests:
+            cpu: 100m
+            memory: 60Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+        ports:
+        - containerPort: 9443
+          name: webhook-server
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /tmp/k8s-webhook-server/serving-certs
+          name: cert
+          readOnly: true
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+      volumes:
+      - name: cert
+        secret:
+          defaultMode: 420
+          secretName: webhook-server-cert
```

Comment:
These names should be more descriptive than "webhook".  This doesn't distinguish it from some other product's label.

And the pod label names, and service account name...secret name, cluster role...

Reply:
If we separated the webhook from nnf-sos, you'd almost certainly attempt to use the same non-descriptive names....

Reply:
All of these names get prefixed with "dws-operator" and they're in the dws-operator-system namespace

Reply:
right, thanks